.bazelrc

```
@@ -138,6 +138,23 @@ build:release-linux-aarch64   --copt=-march=armv8.1a
 
 build:release-mac --config=release-common --platforms=@//tools/platforms:darwin_x86_64
 
+build:release-mac-arm64 --platforms=@//tools/platforms:darwin_arm64
+build:release-mac-arm64 --define release=true
+build:release-mac-arm64 --compilation_mode=opt
+build:release-mac-arm64 --config=backtracesymbols
+build:release-mac-arm64 --config=shared-libs
+build:release-mac-arm64 --config=versioned
```

should this be

```
```suggestion
build:release-mac-arm64 --config=release-common --config=shared-libs --platforms=@//tools/platforms:darwin_arm64
```

to share more code?